blocks/adp-collection-header/adp-collection-header.css

```diff
@@ -77,3 +90,46 @@
     align-items: center;
     justify-content: center;
 }
+
+.toast {
+    display: flex;
+    align-items: center;
+    background-color: #4CAF50; /* Dark green background */
```

Comment:
@TyroneAEM Please use #5258e4 code same to match current theme 
![image](https://github.com/hlxsites/adobe-gmo/assets/61603050/d64f8c12-1548-456c-b019-19b154b55b09)